mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -273,6 +273,8 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_32316 canFreeze ()Z
 	METHOD method_32317 setFrozenTicks (I)V
 		ARG 1 frozenTicks
+	METHOD method_32318 getStackHolder (I)Lnet/minecraft/class_5630;
```

Comment:
I strongly suggest moving this to #1984 as you named this class there. This name imo is debatable.